packages/contracts/contracts/SmartInvoiceEscrowV2.sol

```diff
@@ -0,0 +1,444 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I'm not a fan of breaking git histories via copy-paste. Ideally this should be tracked via a `git mv` to rename the file and updated with the modifications.

Reply:
ok, will do. I had left this in on the interim idea that we might need both versions available, but that doesn't seem likely.